### PR TITLE
[play] Minor fixes to help accessibility

### PR DESCRIPTION
**Pull request by yuvipanda**
I ran the [WAVE Firefox Accessibility checker](https://addons.mozilla.org/en-US/firefox/addon/wave-accessibility-tool/)
on play, and fixed the most obvious full errors.

- Added a <title>
- Set lang attribute
- Added title for kernelpicker

### CODE REVIEW

**Comment by yuvipanda** [comment]
I made up the title  for the page and the kernel selector. Happy to use whatever y'all want!

File: applications/play/components/Main.js [hunk 1]
```diff
@@ -129,6 +129,7 @@ class Main extends React.Component<*, *> {
             rel="prefetch"
             href="https://media.giphy.com/media/xUPGcguWZHRC2HyBRS/giphy.gif"
           />
+          <title>nteract play: Run interactive code</title>
```
Comment (rgbkrk) [inline applications/play/components/Main.js:132]: 👍

File: applications/play/components/kernelUI.js [hunk 2]
```diff
@@ -13,7 +13,12 @@ class KernelSelector extends React.Component<*, *> {
     const { kernelspecs, currentKernel, onChange } = this.props;
     return (
       <form>
-        <select value={currentKernel} onBlur={onChange} onChange={onChange}>
+        <select
+          value={currentKernel}
+          onBlur={onChange}
+          onChange={onChange}
+          title="Active Kernel"
```
Comment (rgbkrk) [inline applications/play/components/kernelUI.js:20]: 👍

File: applications/play/pages/_document.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      </html>
+    );
+  }
+}
+
+MyDocument.childContextTypes = {
+  _documentProps: PropTypes.any
+};
+
+export default MyDocument;
```
Comment (rgbkrk) [inline applications/play/pages/_document.js:38]: Sweet, glad you figured out how to make the base page. 😄

**Comment by rgbkrk** [comment]
Thank you, this is great!

**Comment by yuvipanda** [comment]
Thanks, @rgbkrk!

I want to use play for the Berkeley Data8x course starting mid-May. I'll keep contributing fixes on/off.

**Pull request merged**